.github/workflows/interface-unit-tests.yml

```diff
@@ -131,8 +125,6 @@ jobs:
           - '2'
         python-version:
           - '3.9'
-          - '3.10'
-          - '3.11'
```

Comment:
Do we need to make these CI changes here?

Reply:
No, I was just trying to alleviate the CI, but it seems to be blocking it instead.